test/rewards/AutoCompoundingRewards.ts

```diff
@@ -639,32 +639,50 @@ describe('AutoCompoundingRewards', () => {
                     });
                 });
 
-                describe('setting auto-process count', () => {
-                    it('should revert when a non-admin attempts to set the auto-process count', async () => {
-                        await expect(autoCompoundingRewards.connect(user).setAutoProcessCount(1)).to.be.revertedWith(
-                            'AccessDenied'
+                describe('the number of programs to auto-process the rewards for', () => {
+                    it('should initially be larger than zero', async () => {
+                        expect(await autoCompoundingRewards.autoProcessRewardsCount()).to.be.gt(0);
+                    });
+
+                    it('cannot be set by the admin to zero', async () => {
```

Comment:
should be something like `should revert if set to a 0 value`, otherwise it implies that a non admin can set it to 0

Reply:
fixed